README.md

```diff
@@ -55,3 +55,6 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 
 If you need any help anywhere in the process, you can always ask a question on our [Gitter Chat](https://gitter.im/jboss-outreach/gci).
 
+###Wireframes
+
```

Comment:
Icon and thumbnails both? we will be using thumbnails for students and list for organization. For list we'll be using ag-grid ... please make relevant changes